### PR TITLE
Add QueryPie

### PR DESCRIPTION
**Pull request by aescolastico**
**Important:** *Do not tick a checkbox if you haven’t performed its action.* Honesty is indispensable for a smooth review process.

After making all changes to a cask, verify:

- [x] `brew cask audit --download {{cask_file}}` is error-free.
- [x] `brew cask style --fix {{cask_file}}` reports no offenses.
- [x] There are no [open pull requests](https://github.com/Homebrew/homebrew-cask/pulls) for the same update.
- [x] The submission is for [a stable version](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#stable-versions) or [documented exception](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#but-there-is-no-stable-version).

Additionally, **if adding a new cask**:

- [x] Named the cask according to the [token reference](https://github.com/Homebrew/homebrew-cask/blob/master/doc/cask_language_reference/token_reference.md).
- [x] `brew cask audit --new-cask {{cask_file}}` worked successfully.
- [x] `brew cask install {{cask_file}}` worked successfully.
- [x] `brew cask uninstall {{cask_file}}` worked successfully.
- [x] Checked the cask was not [already refused](https://github.com/Homebrew/homebrew-cask/search?q=is%3Aclosed&type=Issues).
- [x] Checked the cask is submitted to [the correct repo](https://github.com/Homebrew/homebrew-cask/blob/master/doc/development/adding_a_cask.md#finding-a-home-for-your-cask).

### CODE REVIEW

File: Casks/querypie.rb [hunk 1]
```diff
@@ -0,0 +1,12 @@
+cask "querypie" do
+  version "6.2.0"
+  sha256 "22edf316966edb1edf13dc2846714cb4523479298f372b02b0fed00092e550f4"
+
+  # d2f8621kw7pn7s.cloudfront.net was verified as official when first introduced to the cask
```
Comment (miccal) [inline Casks/querypie.rb:5]:
```suggestion
  # d2f8621kw7pn7s.cloudfront.net/ was verified as official when first introduced to the cask
```
[The trailing slash serves a purpose.](https://github.com/Homebrew/homebrew-cask/pull/80988)

File: Casks/querypie.rb [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  url "https://d2f8621kw7pn7s.cloudfront.net/latest/QueryPie-#{version}.dmg"
```
Comment (miccal) [inline Casks/querypie.rb:6]: Please add the following `appcast`:
```
appcast "https://d2f8621kw7pn7s.cloudfront.net/latest/latest-mac.yml"
```

File: Casks/querypie.rb [hunk 3]
```diff
@@ -0,0 +1,12 @@
...
+  name "QueryPie"
+  desc "Most advanced Data Warehouse Client for data analytics teams"
```
Comment (miccal) [inline Casks/querypie.rb:8]:
```suggestion
  desc "Data warehouse client for data analytics teams"
```

**Comment by aescolastico** [comment]
Made suggested changes

**Comment by miccal** [comment]
Thank you for your contribution to `homebrew-cask` @aescolastico.

**Review by miccal — APPROVED** [review]

**Pull request merged**